conf/parallel.conf.js

```diff
@@ -10,13 +10,11 @@ exports.config = {
 
   maxInstances: 10,
   commonCapabilities: {
-    'browserstack.use_w3c': true,
     'bstack:options': {
-      'sessionName': 'parallel_test',
-      'buildName': 'webdriver-browserstack',
-      'projectName': 'Test App',
-      'debug': true,
-    },
+      sessionName: 'parallel_test',
+      buildName: 'browserstack-build-1',
```

Comment:
Is projectName required?

Reply:
not really.... we don't use it kahi.

But maybe we can do it if product wants to do so across langs and frameworks.

Reply:
Like I feel if we add then we can come to know that sessions are getting executed from this repo. Like anyways it will be going in Untitled Project, so not an issue.

Reply:
Makes sense. But do we need to track if an xyz session is being run from a specific repo? If yes, then we can add in a separate PR.

P.S. there are chances that users use the same build and project name across scripts as well.

Reply:
Okay cool. Let's keep it as it now. Later if required we can send a separate PR.